go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/imdario/mergo v0.3.11 // indirect
 	github.com/kong/deck v1.5.1
 	github.com/kong/go-kong v0.16.0
-	github.com/kong/kubernetes-testing-framework v0.0.2
+	github.com/kong/kubernetes-testing-framework v0.0.3
```

Comment:
context: `v0.0.3` added a convience feature to the `kind.Cluster` interface to get the original `*rest.Config` for it so that we can easily generate new clients, including the new typed clients this PR adds.